src/Sylius/Bundle/AdminBundle/templates/shared/helper/table.html.twig

```diff
@@ -53,7 +53,7 @@
                             </button>
                         {% endset %}
                         {% if subactions|trim|length > 0 %}
-                            {{ dropdown({ customTrigger: more_button, content: subactions, direction: 'down-end' }) }}
+                            {{ dropdown.default({ customTrigger: more_button, content: subactions, direction: 'down-end' }) }}
```

Comment:
We currently have `default` and `list` macros, but I see little difference in their purposes, as a dropdown essentially functions as a list. I suggest renaming `default` to something more descriptive.